templates/account/signup.html

```diff
@@ -85,4 +105,41 @@ <h2 class="text-xl text-center">Or Sign Up with Email</h2>
 
   </div>
 
+  <script>
+
+    function toggleButtonsOff(state) {
+      let gitButton = document.getElementById("gitButton");
+      let googleButton = document.getElementById("googleButton");
+      let signupButton = document.getElementById("signup");
+      if (state) {
+        gitButton.classList.add("disabledLink");
+        googleButton.classList.add("disabledLink");
+        signupButton.classList.add("disabledLink");
+        return
+      }
+        gitButton.classList.remove("disabledLink");
+        googleButton.classList.remove("disabledLink");
+        signupButton.classList.remove("disabledLink");
+      return
+    }
+
+    function accepted() {
+      let tosCheckbox = document.getElementById("readTOS");
+      if (tosCheckbox.checked) {
+        toggleButtonsOff(false)
+      } else {
+        toggleButtonsOff(true)
+      }
+    }
```

Comment:
```
function handleTosChange() {
  const tosCheckbox = document.getElementById('readTOS');
  toggleButtons(!tosCheckbox.checked);
}
```
**suggestion (optional):** We could simplify by passing the negation of the `checked` state. Also, changing the function's name to use as an event handler later down the code

Reply:
added